EventFilter/SiPixelRawToDigi/interface/GPUSimpleVector.hpp

```diff
@@ -0,0 +1,72 @@
+//  author: Felice Pantaleo, CERN, 2018
+#ifndef GPU_SIMPLEVECTOR_HPP_
+#define GPU_SIMPLEVECTOR_HPP_
```

Comment:
Could we use the CMS-standard extension `.h`? I guess eventually this file will be moved to some "CUDA utilities" package?